pkg/resources/sku/sku.go

```diff
@@ -159,12 +161,18 @@ func (p AProductConfig) Configure(obj metav1.Object) error {
 		replicas = t.Spec.Replicas
 		podTemplate = &t.Spec.Template
 		break
-
+	case *keycloak.Keycloak:
+		if p.sku.isUpdated || t.Spec.Instances < int(configReplicas) {
+			t.Spec.Instances = int(configReplicas)
+		}
+		resources := p.resourceConfigs[KeycloakName].Resources
+		p.mutateResources(t.Spec.KeycloakDeploymentSpec.Resources.Requests, resources.Requests)
+		p.mutateResources(t.Spec.KeycloakDeploymentSpec.Resources.Limits, resources.Limits)
```

Comment:
Doing a fresh install, `t.Spec.KeycloakDeploymentSpec.Resources.Requests` seems to be nil here ? :thinking: